examples/api/lib/material/dropdown_menu/dropdown_menu.2.dart

```diff
@@ -39,6 +39,9 @@ class _DropdownMenuExampleState extends State<DropdownMenuExample> {
   @override
   Widget build(BuildContext context) {
     final ColorScheme colorScheme = Theme.of(context).colorScheme;
+    final List<DropdownMenuEntry<String>> menuEntries = list.map<DropdownMenuEntry<String>>((String value) {
+      return DropdownMenuEntry<String>(value: value, label: value);
+    }).toList();
```

Comment:
```suggestion
  static final List<MenuEntry> menuEntries = UnmodifiableListView<MenuEntry>(
    list.map<MenuEntry>((String name) => MenuEntry(value: name, label: name)),
  );

  @override
  Widget build(BuildContext context) {
    final ColorScheme colorScheme = ColorScheme.of(context);
```

Reply:
Good catch! The intention was to move it above the build like the other examples.